frontend/src/components/regions/CardDetails/ReviewPopUp.jsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+
+import Markdown from "react-markdown";
+import Box from "@material-ui/core/Box";
+import Modal from "../../widgets/Modal";
+import Card from "@material-ui/core/Card";
+import { makeStyles } from "@material-ui/core/styles";
+import ReviewStatus from "../../widgets/ReviewStatus";
+import { CardActions, CardContent, Typography } from "@material-ui/core";
+import Grid from "@material-ui/core/Grid";
+import Paper from "@material-ui/core/Paper";
+
+const useStyles = makeStyles((theme) => ({
+  cardStyle: {
+    display: "block",
+    position: "relative",
+    maxWidth: "90%",
+  },
+  timeStyle: {
+    fontSize: 11,

```

Comment:
Please make the fontSize to be as you have done for your `CardContent` class, 11px

Reply:
adressed